src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.cs

```diff
@@ -35,7 +35,7 @@ public Stream CreateReadStream()
                 FileMode.Open,
                 FileAccess.Read,
                 FileShare.ReadWrite,
-                1024 * 64,
+                Math.Min((int)Length, 1024 * 64),
```

Comment:
`FileStream` defines the default buffer size to be 4k. Any idea why we picked a significantly larger number? Does it make sense to stick to the 4k value?